pkg/controller/vpcpeering/peering.go

```diff
@@ -284,15 +314,17 @@ func (e *external) Update(ctx context.Context, mg cpresource.Managed) (managed.E
 	}
 
 	if !routeTableReady && cr.Status.AtProvider.VPCPeeringConnectionID != nil {
+		vpcIDs := []string{*cr.Spec.ForProvider.VPCID}
+		if isInternal {
+			vpcIDs = append(vpcIDs, *cr.Spec.ForProvider.PeerVPCID)
+		}
 		filter := ec2.Filter{
-			Name: aws.String("vpc-id"),
-			Values: []string{
-				*cr.Spec.ForProvider.VPCID,
-			},
+			Name:   aws.String("vpc-id"),
+			Values: vpcIDs,
 		}
 		describeRouteTablesInput := &ec2.DescribeRouteTablesInput{
 			Filters:    []ec2.Filter{filter},
-			MaxResults: aws.Int64(10),
+			MaxResults: aws.Int64(100),
```

Comment:
why need change to 100 ？

Reply:
There are already 5 routing tables now, so give a larger value